### PR TITLE
Support for 3d primitive solid shape; fixes #10

### LINKED ISSUE #10: DST-CAT-SR-0060: Support for 3D Primitive Solid Shapes
The mapping shall at least support the following 3D primitive solid shapes: rectangular box, cylinder, cone, sphere, paraboloid, tetrahedron, wedge, capsule.

### PR DESCRIPTION
**Pull request by nathanatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/DEH-CATIA/pulls) open
- [x] I have verified that I am following the DEH-CATIA [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/DEH-CATIA/master/.github/CONTRIBUTING.md)
- [ ] I have provided test coverage for my change (where applicable)

### Description

- [ ] adding support for 3d solid shape

### CODE REVIEW

**Comment by nathanatrhea** [comment]
image

File: DEHCATIA/MappingRules/CatiaProductToElementRule.cs [hunk 1]
```diff
@@ -371,6 +394,57 @@ private void MapParameter(ParameterType parameterType, string parameterTypeShort
...
+                        }
+                    }
+                };
+
+                definitionRow.ElementDefinition.Parameter.Add(parameter);
+            }
+
+            this.UpdateValueSet(parameter, values);
+
+            Logger.Info($"The {parameterTypeShortName}  parameter has been updated for the {definitionRow.Name}");
```
Comment (alexatrhea) [inline DEHCATIA/MappingRules/CatiaProductToElementRule.cs:445]: I think double space here in this string.

File: DEHCATIA/Services/ParameterTypeService/IParameterTypeService.cs [hunk 2]
```diff
@@ -61,6 +61,46 @@ public interface IParameterTypeService
...
 
+        /// <summary>
+        /// The Shape Kind <see cref="ParameterType"/>
+        /// </summary>
+        ParameterType ShapeKind { get; }
+
+        /// <summary>
+        /// The Shape length <see cref="ParameterType"/>
+        /// </summary>
+        public ParameterType ShapeLength { get; }
```
Comment (alexatrhea) [inline DEHCATIA/Services/ParameterTypeService/IParameterTypeService.cs:72]: This is an interface so "public" does not have to be declared.

**Review by alexatrhea — APPROVED** [review]
Approved pending small errors.

**Pull request merged**